src/wikitextprocessor/core.py

```diff
@@ -1410,7 +1410,14 @@ def expander(arg: str) -> str:
                         continue
 
                     # Construct and expand template arguments
-                    self.expand_stack.append(name)
+                    self.expand_stack.append("Template:" + name)
+                    if detect_expand_template_loop(self.expand_stack):
+                        parts.append(
```

Comment:
Should we have a `wxr.wtp.error` or `wxr.wtp.warning` here? We are handling the issue, and it's caused by bad code on Wiktionary's side, so not necessarily.

Reply:
I'll add a warning here.